tutorials/editor/using_the_xr_editor.rst

```diff
@@ -0,0 +1,64 @@
+.. _doc_using_the_xr_editor:
+
+Using the XR editor
+===================
+
+In 2024, we introduced the `Godot XR Editor <https://godotengine.org/article/godot-editor-horizon-store-early-access-release/>`__, a version of the Godot Editor **designed to run natively on XR devices**, enabling the creation, development and export of 2D, 3D and **XR** apps and games directly on device.
+
+The app can be downloaded from the `Meta Horizon Store <https://www.meta.com/experiences/godot-game-engine/7713660705416473/>`__, or from the `Godot download page <https://godotengine.org/download/preview/>`__.
+
+.. note::
+
+    The XR editor is in early access, while we continue to refine the experience. See :ref:`doc_using_the_xr_editor_limitations` below.
+
+XR devices support
+------------------
+
+For now, the Godot XR Editor is only available for the following `Meta Quest <https://www.meta.com/quest/>`__ devices running **Meta Horizon OS v69 or higher**:
```

Comment:
```suggestion
For now, the Godot XR Editor is only available for the following `Meta Quest <https://www.meta.com/quest/>`__
devices running **Meta Horizon OS v69 or higher**:
```

Reply:
Should the word editor in "XR Editor" be capitalized in the docs? The other writings of it here have it not capitalized.